esp/src/eclwatch/templates/CurrentUserDetailsWidget.html

```diff
@@ -27,16 +29,22 @@ <h2>
                     </li>
                     <li>
                         <label class="Prompt" for="${id}OldPassword">${i18n.OldPassword}:</label>
-                        <input name="oldPassword" type="password" required="true" data-dojo-type="dijit.form.ValidationTextBox" />
+                        <input name="oldPassword" type="password" required="true"
+                            data-dojo-type="dijit.form.ValidationTextBox" />
                     </li>
-                    <div name="newPassword" data-dojo-props="required: false" data-dojo-type="dojox.form.PasswordValidator">
+                    <div name="newPassword" data-dojo-props="required: false"
+                        data-dojo-type="dojox.form.PasswordValidator">
                         <li>
                             <label class="Prompt" for="${id}NewPassword">${i18n.NewPassword}:</label>
-                            <input name="newPassword" type="password" required="true" pwtype="new" data-dojo-props="invalidMessage:'${i18n.PasswordsDoNotMatch}', placeHolder:'${i18n.MustContainUppercaseAndSymbol}'" data-dojo-type="dijit.form.ValidationTextBox" />
+                            <input name="newPassword" type="password" required="true" pwtype="new"
+                                data-dojo-props="invalidMessage:'${i18n.PasswordsDoNotMatch}', placeHolder:'${i18n.MustContainUppercaseAndSymbol}'"
+                                data-dojo-type="dijit.form.ValidationTextBox" />
                         </li>
                         <li>
                             <label class="Prompt" for="${id}VerifyPassword">${i18n.ConfirmPassword}:</label>
-                            <input name="newPasswordRetype" type="password" required="true" pwtype="verify" data-dojo-props="invalidMessage:'${i18n.PasswordsDoNotMatch}', placeHolder:'${i18n.MustContainUppercaseAndSymbol}'" data-dojo-type="dijit.form.ValidationTextBox" />
+                            <input name="newPasswordRetype" type="password" required="true" pwtype="verify"
+                                data-dojo-props="invalidMessage:'${i18n.PasswordsDoNotMatch}', placeHolder:'${i18n.MustContainUppercaseAndSymbol}'"
+                                data-dojo-type="dijit.form.ValidationTextBox" />
```

Comment:
Looks like you have enabled a HTML formatter?  Can you roll back these changes (or adjust your formatter to not split the lines?